crates/core_arch/src/powerpc/altivec.rs

```diff
@@ -368,21 +368,14 @@ unsafe extern "C" {
     #[link_name = "llvm.ppc.altivec.srv"]
     fn vsrv(a: vector_unsigned_char, b: vector_unsigned_char) -> vector_unsigned_char;
 
-    #[link_name = "llvm.ctlz.v16i8"]
-    fn vclzb(a: vector_signed_char) -> vector_signed_char;
-    #[link_name = "llvm.ctlz.v8i16"]
-    fn vclzh(a: vector_signed_short) -> vector_signed_short;
-    #[link_name = "llvm.ctlz.v4i32"]
-    fn vclzw(a: vector_signed_int) -> vector_signed_int;
-
     #[link_name = "llvm.ppc.altivec.vrlb"]
     fn vrlb(a: vector_signed_char, b: vector_unsigned_char) -> vector_signed_char;
     #[link_name = "llvm.ppc.altivec.vrlh"]
     fn vrlh(a: vector_signed_short, b: vector_unsigned_short) -> vector_signed_short;
     #[link_name = "llvm.ppc.altivec.vrlw"]
     fn vrlw(a: vector_signed_int, c: vector_unsigned_int) -> vector_signed_int;
 
-    #[link_name = "llvm.ppc.altivec.vrfin"]
+    #[link_name = "llvm.nearbyint.v4f32"]
     fn vrfin(a: vector_float) -> vector_float;
```

Comment:
I've verified that these in fact produce identical assembly https://godbolt.org/z/Wx1KWezbe